k8s/prod/common/rse/check.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
You cannot deploy to prod without an ATO. I know this is just a test, not sure though.

Reply:
Yes. I don't think I really need to go to prod. I will remove this bit.